python/paddle/fluid/layers/tensor.py

```diff
@@ -600,6 +605,9 @@ def assign(input, output=None):
                 value_name: values
             })
 
+    if is_inplace and in_dygraph_mode():
+        output._bump_inplace_version()
+
```

Comment:
Why need to change the version in the python side?

Reply:
For Python api `assign(input, output=None)`,  it can only be checked on the python side that whether `output` is changed inplace. And when changed inplace,  the inplace  version of `output` should be increased by 1.